tests/e2e/memgraph.py

```diff
@@ -124,7 +125,10 @@ def start(self, restart=False, args=None, setup_queries=None):
             "300",
             "--storage-properties-on-edges",
         ] + self.args
-        self.bolt_port = extract_bolt_port(args_mg)
+        if bolt_port:
+            self.bolt_port = bolt_port
+        else:
+            self.bolt_port = extract_bolt_port(args_mg)
         self.proc_mg = subprocess.Popen(args_mg)
```

Comment:
Does it make sense to have both here if else which calls extract_bolt_port and in the caller? Maybe just choose between caller and callee (I would do in callee logic, I think is generally better)